cogs/roll.py

```diff
@@ -0,0 +1,40 @@
+"""Command to imitate an ingame dice roll. Can roll randomly, or return the summary of the roll."""
+from hagadias.helpers import DiceBag
+import logging
+
+from discord import Colour, Embed
+from discord.ext.commands import Cog, Context, command
+
+log = logging.getLogger('bot.' + __name__)
+
+
+class Roll(Cog):
+    """Takes a dice string argument and either returns the stats for it or use as a base to roll.
+    Syntax: ?dice (dice string)"""
+
+    @command()
+    async def dice(self, ctx: Context, *args):
+        """Returns the (min, max) Average (average)."""
+        log.info(f'({ctx.message.channel}) <{ctx.message.author}> {ctx.message.content}')
+        val = ' '.join(args)
+        avg = DiceBag(val).average()
+        min = DiceBag(val).minimum()
+        max = DiceBag(val).maximum()
```

Comment:
`min` and `max` are the names of Python builtin functions, so usually we don't want to override those names - could they be called something else?

Reply:
Slipped my mind. Will be renaming these variables to minval, maxval, and avg to avgval to unify the name scheme.